### DISCUSSION TITLE
Laravel 10

### DISCUSSION THREAD

**Discussion by jcre** [spatie/laravel-backup-server#51] [Q&A]
Hi,

I'm trying to set this up in a brand new Laravel 10 project but I keep running into a problem:

```
  Problem 1
    - Only one of these can be installed: illuminate/notifications[v5.8.0, ..., v5.8.36, v6.0.0, ..., v6.20.44, v7.0.0, ..., v7.30.6, v8.0.0, ..., v8.83.27, v9.0.0, ..., v9.52.5, v10.0.0, ..., v10.5.1], laravel/framework[v10.0.0, ..., v10.5.1]. laravel/framework replaces illuminate/notifications and thus cannot coexist with it.
```

Is this something I'm doing wrong, or is this package not yet Laravel 10 compatible? 

Thanks!

**Comment by freekmurze** [comment]
We indeed didn't release a L10 version yet...

I've just taken care of it, you should now be able to install it in a Laravel 10 app.
